libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxVoiceInstructionsPlayer.kt

```diff
@@ -74,6 +74,15 @@ class MapboxVoiceInstructionsPlayer @JvmOverloads constructor(
             play()
         }
 
+    /**
+     * Change language in runtime.
+     *
+     * @param language the new [Locale] language (in a format acceptable by [Locale])
+     */
+    fun updateLanguage(language: String) {
```

Comment:
```suggestion
    fun updateLanguage(language: String): Boolean {
```

Curious if there should be some feedback here on whether the language update was successful

Reply:
but then again, this may not be best because there are a few ways the language could fail. Boolean result is not very informative, we could add a callback to get real status some time later.